download-scryfall-cards.mjs

```diff
@@ -107,8 +107,14 @@ const stripped = cards.filter(card => {
         isDigital: card.digital,
         isPromo: !customNotPromoSets.includes(card.set) && (card.promo || card.promo_types || customPromoSetTypes.includes(card.set_type) || customPromoSets.includes(card.set)),
         imageUris: {
-            front: `https://api.scryfall.com/cards/${card.set}/${card.collector_number}?format=image&version=border_crop&face=front`,
-            back: card.card_faces?.[1]?.image_uris ? `https://api.scryfall.com/cards/${card.set}/${card.collector_number}?format=image&version=border_crop&face=back` : undefined,
+            front: {
+                cropped: `https://api.scryfall.com/cards/${card.set}/${card.collector_number}?format=image&version=border_crop&face=front`,
+                normal: `https://api.scryfall.com/cards/${card.set}/${card.collector_number}?format=image&version=normal&face=front`,
```

Comment:
These endpoints only differ by a query param, so maybe instead of storing the full url here it's easier to just modify that query param where it's used?

Reply:
I'm writing up a quick commit to maybe switch to that. Just realized that I don't have node installed on this computer though so gonna take me a few minutes longer than expected >.>

Reply:
Here's a trimmed down commit that _just_ tackles switching between the Border Crop and Normal images.

It doesn't
- Fix the card count.
- Re-add your feature to print non-dfc card backs.
- Update the print scale.
- Rearrange the configuration.

Reply:
Added another commit, takes care of configuring Card Backs and fixes the count behavior.

Reply:
![Screenshot 2024-07-24 at 10 31 54 PM](https://github.com/user-attachments/assets/8a4b8289-aef6-4a71-b9f8-805f485f409c)

Would something like this work for now? 
Just added "Actual" as a quick option that has the sizing you want.